### PR TITLE
docs - restructure the platform requirements extensions table

### PR DESCRIPTION
**Pull request by lisakowen**
refactor the extensions table in the platform requirements topic.  in this PR:
- get rid of most of the superscripts and notes by making the extension name in the table an xref.
- identify package versions AND identify additional version info for java and the PL/s, like what version of the language is supported/installed with the package or in the image.

doc review site link (will need to scroll up a little):
- http://docs-lisa-platreq-exts.cfapps.io/7-0/install_guide/platform-requirements.html#topic_eyc_l2h_zz

for comparison, link to table in 6.7 docs (again, scrolling up is required):
- https://gpdb.docs.pivotal.io/6-7/install_guide/platform-requirements.html#topic_eyc_l2h_zz

### CODE REVIEW

**Comment by lisakowen** [comment]
@HaozhouWang - can you please verify that the R/Python versions i added to the table for the PL/s are correct?

**Comment by lisakowen** [comment]
@dyozie - thx for reviewing, i updated madlib/postgis versions.

not sure what you mean by "list the container versions for pl/container beta"?  the table includes a row for the R image for pl/c 3 beta.  do you want the docker version listed?  or?

File: gpdb-doc/dita/install_guide/platform-requirements.xml [hunk 1]
```diff
@@ -322,70 +322,93 @@
...
-            format="html" scope="external">MADlib FAQ</xref>. <p><sup>2</sup>PL/R supports R 3.5.1.
-            On RHEL and CenOS the PL/R package installs R 3.3.3. See <xref
-              href="../analytics/pl_r.xml" format="dita" scope="peer">PL/R
-              Language</xref>.</p><p><sup>3</sup>For information about the Python package, including
-            the modules provided, see the <xref href="../install_guide/install_python_dsmod.xml"
-              format="dita" scope="peer">Python Data Science Module Package</xref>.
-              </p><p><sup>4</sup>For information about the R package, including the libraries
-            provided, see the <xref href="../install_guide/install_r_dslib.xml" format="dita"
-              scope="peer">R Data Science Library Package</xref>. </p></note>
+            format="html" scope="external">MADlib FAQ</xref>.</note>
```
Comment (lenapivotal) [inline gpdb-doc/dita/install_guide/platform-requirements.xml:410]: I wonder if this note could go in the "additional information" column for Madlib. I can see you're adding versions there but it we could just say "support matrix at MADlib FAQ"....

Comment (lisakowen) [reply gpdb-doc/dita/install_guide/platform-requirements.xml:410]: great idea!  will do ...

**Comment by dyozie** [comment]
> @dyozie - thx for reviewing, i updated madlib/postgis versions.
> 
> not sure what you mean by "list the container versions for pl/container beta"? the table includes a row for the R image for pl/c 3 beta. do you want the docker version listed? or?

Ah - you're right.  Please disregard that comment.

**Comment by HaozhouWang** [comment]
LGTM, just minor comment:
1. GreenplumR has GA now.
2. PL/Container version -> 2.1.2
3. PL/Container R image -> 2.1.2 with R 3.6.3

**Comment by lisakowen** [comment]
i updated those versions in https://github.com/greenplum-db/gpdb/pull/10027/commits/6be562c7b355aaed0c8710ec92e8162746a8a08d, but forgot to re-push the staging site ...  thanks for reviewing, @HaozhouWang .

**Pull request merged**